Start of navbar issue #85 - Add blank navbar

kfili opened the pull request.

Commit:
Add blank navbar to rails app

Commit:
Remove unusable 'home' button from navbar

kfili commented:
This PR will had a black bar to the top of the rails app so it is presentable enough I think. And shouldn't add any bugs.

mzagaja commented:
This navbar seems to have some white space above it. Is that intentional?
<img width="1279" alt="screen shot 2017-06-27 at 7 42 05 pm" src="https://user-images.githubusercontent.com/565647/27614611-bbd14258-5b70-11e7-9f18-27eab1d459a7.png">

Commit:
Add links to Navbar

kfili commented:
@AlexMerritt @mzagaja 
I moved the navbar up so there no more white space above it.
I also added links to the navbar so it can now be used to navigate to all pages - at least all pages that there are links to from the "home" page - "home" "All listings" and "discrimantory listings"
